Test of extension Interpolation fails with NPE

## Setup
SUT: http://conformance.rasdaman.com/ows?service=WCS&request=GetCapabilities&acceptversions=2.0.1
Selected CCs: only `interpolation`
Tested locally with docker environment 
  * ETS WCS 2. 0 1.13-SNAPSHOT (includes PR #48)
 * TEAM Engine 5.2

## Testfailure
" Error in call to extension function {public org.w3c.dom.NodeList com.occamlab.te.TECore.request(org.w3c.dom.Document,java.lang.String) throws java.lang.Throwable}: Exception in extension function java.lang.reflect.InvocationTargetException"
 *  Failure from error log: 
```
Caused by: java.lang.NullPointerException
        at com.occamlab.te.TECore.updateParentTestResult(TECore.java:1051)
        at com.occamlab.te.TECore.callTest(TECore.java:1024)
        at sun.reflect.GeneratedMethodAccessor42.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at net.sf.saxon.functions.ExtensionFunctionCall.invokeMethod(ExtensionFunctionCall.java:533)
        at net.sf.saxon.functions.ExtensionFunctionCall.call(ExtensionFunctionCall.java:256)
        at net.sf.saxon.functions.ExtensionFunctionCall.iterate(ExtensionFunctionCall.java:147)
        ... 76 more
```

--- a/src/main/scripts/ctl/ext_get-int.xml
+++ b/src/main/scripts/ctl/ext_get-int.xml
@@ -333,7 +333,7 @@
 		</ctl:code>
 	</ctl:test>
 	<!--req8-->
-	<ctl:test name="wcs2:get-int-req8">
+	<ctl:test name="wcs2:get-int-req8-post">
 		<ctl:param name="url"/>
 		<ctl:param name="Capabilities"/>
 		<ctl:param name="DescribeCoverage"/>
@@ -388,7 +388,7 @@
 		</ctl:code>
 	</ctl:test>
 	<!--req9-->
-	<ctl:test name="wcs2:get-int-req9">
+	<ctl:test name="wcs2:get-int-req9-soap">
 		<ctl:param name="url"/>
 		<ctl:param name="Capabilities"/>
 		<ctl:param name="DescribeCoverage"/>

